control_panel_api/k8s.py

```diff
@@ -0,0 +1,60 @@
+from django.http import HttpResponse
+from kubernetes import client, config
+from kubernetes.config.config_exception import ConfigException
+import requests
+
+
+def proxy(request):
+    return Request(request).make()
+
+
+class Request(object):
+
+    def __init__(self, request):
+        self.request = request
+        self._config = Config()
```

Comment:
This will load the config on every request - that seems inefficient. Shouldn't we load the config once on startup?

Reply:
You might want to pass the config in as a constructor parameter, then you can pass a test config object instead of monkey-patching the Config class in the tests.

Reply:
Yes good point, I'll play with it any see if I can move to the settings.

Reply:
Also, I need to check if the authentication token changes during the lifecycle of the pod running the CP-API.

Reply:
FYI: Had a chat with Kerin and I should be able to cache the token and config at start 👍